client/src/views/Home.tsx

```diff
@@ -206,28 +204,18 @@ function HeaderContentRight(): JSX.Element {
 }
 
 function Home(): JSX.Element {
-  const ASSETS_API_URL = `${APP_API_BASE_URL}/assets`;
   const classes = useStyles();
   const [donations, setDonations] = React.useState<Asset[]>([]);
   const [requests, setRequests] = React.useState<Asset[]>([]);
 
   React.useEffect(() => {
     // fetch assets with querySearchText
-    const assetsApiDonate = new URL(ASSETS_API_URL);
-    assetsApiDonate.searchParams.append('type', 'donation');
-    assetsApiDonate.searchParams.append('limit', '3');
-    assetsApiDonate.searchParams.append('offset', '0');
-    fetch(assetsApiDonate.href)
+    fetch('http://localhost:3001/api/assets?type=donation&limit=3&offset=0')
       .then((resp) => resp.json())
       .then((data: Asset[]) => {
         setDonations(data);
       });
-
-    const assetsApiRequest = new URL(ASSETS_API_URL);
-    assetsApiRequest.searchParams.append('type', 'request');
-    assetsApiRequest.searchParams.append('limit', '3');
-    assetsApiRequest.searchParams.append('offset', '0');
-    fetch(assetsApiRequest.href)
+    fetch('http://localhost:3001/api/assets?type=request&limit=3&offset=0')
       .then((resp) => resp.json())
       .then((data: Asset[]) => {
         setRequests(data);
```

Comment:
Can you use the same url that was in the original code? That will ensure that it works in staging as well. Also, instead of passing static strings as the query params, can you use the existing example?

Reply:
@esteban-gs pushed the update as requested~